src/VirtoCommerce.XCatalog.Core/Schemas/PropertyGroupType.cs

```diff
@@ -0,0 +1,44 @@
+using GraphQL.Types;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.Xapi.Core.Extensions;
+using VirtoCommerce.Xapi.Core.Schemas;
+
+namespace VirtoCommerce.XCatalog.Core.Schemas
+{
+    public class PropertyGroupType : ExtendableGraphType<PropertyGroup>
+    {
+        public PropertyGroupType()
+        {
+            Name = "PropertyGroup";
+            Description = "Property group.";
+
+            Field(x => x.Id, nullable: false).Description("The unique ID of the property group.");
+            Field(x => x.Name, nullable: false).Description("The name of the property group.");
+            Field(x => x.Priority, nullable: true).Description("The display order of the property group.");
+
+            Field<StringGraphType>("localizedName").Resolve(context =>
```

Comment:
Should be only name + description (that resolve value from name + localizedName and localizedDescription)